config/routes.rb

```diff
@@ -21,7 +21,7 @@
     delete '/delete_user/:id', to: 'update_user#destroy_user', as: :delete_user
   end
 
-  resources :users, only: [:edit, :update, :create]
+  resources :users, only: [:edit, :update, :create, :show]
```

Comment:
I think it makes most sense for users#show to be the settings page that updates things. Profile pages should be placed under the role namespaces. For example, a user might be a professional and a requester. They will have two profile pages namespaced under those roles. The profile pages will display different information. This also makes me realize that we will need to store ratings that are tied to a role. 😱